### ISSUE TITLE
Add timestamp to host_vehicle_location

### ISSUE THREAD

**Issue by u21n80** [OpenSimulationInterface/open-simulation-interface#463]
Hi,

currently there is now no timestamp associated to the host_vehicle_location. For completeness this should be added since other fields like the moving objects or lane have a separate header that contains the timestamp of the related signals. If there is no timestamp for the host_vehicle_location it is impossible to determine, what the time of the last update was.

**Comment by ThomasSchloemicherAVL** [comment]
Please check if it is already covered in PR#441.
@ThomasNaderBMW FYI

**Comment by ThomasNaderBMW** [comment]
@u21n80 : Could you please give some feedback in which message you would like to see the timestamp?
PR #441 .

**Comment by jruebsam** [comment]
Hi, 

I would say  we could just put it into the  message root as timestamp. 
I am not sure however, if this message contradicts the last_measurement_time message in Sensordata.

**Comment by u21n80** [comment]
@ThomasNaderBMW I agree with @jruebsam I would also put it in the root message of HostVehicleData.

**Comment by pmai** [comment]
I think there are multiple uses of the host vehicle location across OSI with different semantics which will likely have to have their own last measurement times: The location contained in HostVehicleData is the location as coordinated by the overall position estimation function of the vehicle, and would thus be tagged with the time it was calculated and communicated across the car networks (which might be the same for all values in HostVehicleData, in which case an overall time-stamp would be sufficient).

Individual position sensors (IMU, GPS, Sensor with position estimation capabilities, ...), which are currently half-way supported by SensorData (this should be improved) might have their own measurement time stamps, and those should be communicated in their respective fields. I.e. while driving through a tunnel I would assume the data in HostVehicleData (both cartesian and geo referenced) to be continually updated based on IMU GPS fusion, but the data in a SensorData (or similar message) coming from a GPS receiver to not update/indicate lost transmission (trivialized example here, GPS sensor might be internally updating based on internal IMU/position estimation, but this should be indicated in its output).
